harness/determined/cli/_color_json.py

```diff
@@ -0,0 +1,92 @@
+"""
+colorize JSON output.
+test string:
+   echo '{"a": "b", "c": ["1", 2.0, 3.1, 5, "4", {"d": true, "e": null, "f": [1, 2, 3]}]}'
+"""
+
+import json
+from typing import Any, TextIO
+
+from termcolor import colored
```

Comment:
[google style guide for python imports](https://google.github.io/styleguide/pyguide.html#22-imports)